build/configs/artik053/README.md

```diff
@@ -91,41 +91,24 @@ openocd -f artik053.cfg -c ' \
 
 ## ROMFS
 
-Before executing below steps, execute [generic steps](../../../tools/fs/README_ROMFS.md), step 1 and step 2.
+Execute [generic steps](../../../tools/fs/README_ROMFS.md), step 1 and step 2.
+If you want to change the size of ROMFS partition, execute below steps.
 
-3. Modify partition configs.  
-    Below steps creates ROMFS partition with size 400KB at next of user partition.  
-    1. Split user partition size from (1400) to (1000, 400) in ARTIK053_FLASH_PART_LIST
+1. Modify partition configs.
+    By default, ROMFS partition with size 400KB is set at the next of user partition (with size 1000KB).
```

Comment:
1. No, only your defconfig for iotivity has it.
Default values are set in [Kconfig](https://github.com/Samsung/TizenRT/blob/master/os/arch/arm/src/artik053/Kconfig) on artik053.
Search *ARTIK053_FLASH_PART_LIST* at above link.
2. Defaultly, we don't need ROMFS. I don't agree that we should set it enable.
We need more discussion.
3. same for type and name.

Reply:
Ok. I modify it. ROMFS is needed defaultly when we use artik053/iotivity config.

Reply:
Ok. I modify it. ROMFS is needed defaultly when we use artik053/iotivity config.